spec/truffle/interop/java_new_spec.rb

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+require_relative '../../ruby/spec_helper'
+
+guard -> { !Truffle.native? } do
+  describe "java_class.new" do
+
+    it "creates basic objects" do
+      big_integer = Truffle::Interop.java_type("java.math.BigInteger").new("14")
+      Truffle::Interop.java_type_name(big_integer.getClass!).should == "java.math.BigInteger"
+      big_integer.intValue!.should == 14
```

Comment:
Side note: this illustrates why `intValue` sending READ is not nice and counter-intuitive.

Reply:
Yes I already mentioned that this PR exposes some bad interop design. We'll address that separately - it wasn't introduced in this PR.